promet/nclex
# NCSBNSB-10:  Rectify status report issues

This PR rectifies the following issues that appear in the status report:

* Configure trusted hosts
* Remove the following modules from the site:
  - checklistapi
  - hacked
  - security review
  - seo checklist

This PR also contains changes to the Vagrantfile for the local developer
build to implement a separate provisioner that runs several drupal
provisioning steps (drush cim, drush updb, drush entup, drush cron, drush cr)

To test:

Navigate to /admin/reports/status:

* Trusted Hosts patterns should reflect that they are configured

Navigate to /admin/modules:

* Checklist API, Hacked, Securty Review and SEO Checklist should no longer appear on the module listing

branch NCSBNSB-10-rectify-status-report-issues
